Sources/Common/Core/DataArray/test/testDataArray.js

```diff
@@ -202,6 +235,47 @@ test('Test vtkDataArray getRange function with multi-channel data.', (t) => {
   t.end();
 });
 
+test('Test vtkDataArray getRanges function with multi-channel data.', (t) => {
+  // create a data array with 3 channel data.
+  const newArray = new Uint16Array(256 * 3);
```

Comment:
there is no need to make that array that large. 
If you make it up tot 3*10, the magnitude will be a round number and you wouldn't need the convenient `compareFloat` function

Reply:
I can't say I understand how this magnitude is calculated. I just tried an array of 3 * 10, but the maximum of the magnitude was 33.67491648096547.  I tried with other small ranges (3*2 - the max is 3.741657386773941) and wasn't able to get the max to be a whole number.

Reply:
indeed, that won't work with 3 channels, it will work with 4 channels though.